indexer/api/handlers/deposits.go

```diff
@@ -34,11 +35,17 @@ func newDepositResponse(deposits []*database.L1BridgeDepositWithTransactionHashe
 		item := DepositItem{
 			Guid: deposit.L1BridgeDeposit.TransactionSourceHash.String(),
 			Tx: Transaction{
-				BlockNumber:     420420,  // TODO
-				BlockHash:       "0x420", // TODO
-				TransactionHash: "0x420", // TODO
+				// BlockNumber:     420420,  // TODO
+				// BlockHash:       "0x420", // TODO
```

Comment:
THese are in db but frontend doesn't use them (I think?) so just kinda not worrying about them for now